Sources/Widgets/Widgets3D/ShapeWidget/example/index.js

```diff
@@ -172,7 +173,7 @@ reader
     // default slice orientation/mode and camera view
     const sliceMode = vtkImageMapper.SlicingMode.K;
     image.imageMapper.setSlicingMode(sliceMode);
-    image.imageMapper.setSlice(0);
+    image.imageMapper.setSlice(1);
```

Comment:
Do we have a bug here? Shouldn't the initial index by 0 for image mapper?

Reply:
Yes there is a problem here. The scroll bar allows user to browse from slice 1 to N. So if you place something on slice 0. it wont be displayed if you change slices and go back to the 1st slice.

Reply:
Can we fix the scroll bar instead so that it goes between `0` and `N-1`?
```html
<input type="range" min="-1" max="$(N-1)" ...>
```
or at least converting the `[1;N]` range to `[0;N-1]` before setting it on the mapper? Because `setSlice(N)` shouldn't work either.

Reply:
the max of the slice range is set [here](https://github.com/Kitware/vtk-js/blob/49a5ec41c53b6b4035a208620f26254820ae5cb2/Sources/Widgets/Widgets3D/ShapeWidget/example/index.js#L140), so i think the problem is solved just by setting the range min to 0

Reply:
Or to `extent[slicingMode * 2]` (does not technically have to be 0) 🙂

Reply:
> does not technically have to be 0
Not sure i understood you correctly here. You refer to the max or minimum value when you mention the 0? Because i think the minimum should be 0 on every minimum slice, since the example is using the slice to place the point in space, so the image mapper would'nt be a cube when extended to 3D

Reply:
@ocrossi you might want to consider the following: https://vtk.org/Wiki/VTK/Tutorials/Extents